docs/src/ZIndex Classes.doc.js

```diff
@@ -1,181 +1,660 @@
 // @flow strict
 import React, { type Node } from 'react';
-import { Box, Flex, Icon, Text } from 'gestalt';
-import Example from './components/Example.js';
-import Card from './components/Card.js';
 import PageHeader from './components/PageHeader.js';
-import Markdown from './components/Markdown.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
+card(<FeedbackCallout componentName="zIndex Classes" />);
+
 card(
   <PageHeader
     name="zIndex Classes"
     fileName="zIndex"
-    description="FixedZIndex and CompositeZIndex are 2 classes that generate z-index for the Box and Sticky components."
+    description="FixedZIndex and CompositeZIndex are utility classes that generate z-indexes for Gestalt components."
   />,
 );
 
 card(
-  <Card name="FixedZIndex">
-    <Flex alignItems="start" direction="column" gap={4}>
-      <Text>
-        FixedZIndex is used for setting fixed z-index values. FixedZIndex must be instantiated with
-        a number.
-      </Text>
-      <Box padding={2} color="lightGray" rounding={2}>
-        <Markdown
-          text="
+  <MainSection name="FixedZIndex">
+    <MainSection.Subsection columns={2}>
```

Comment:
Can/should these be full-width?

Reply:
YEP! good catch

DONE

Reply:
we have a max-width on markdown, so this is expected